src/UseCases/ValidateMembershipFee/ValidateMembershipFeeUseCase.php

```diff
@@ -6,6 +6,9 @@
 
 use WMDE\Euro\Euro;
 
+/**
+ * @deprecated due to the payment refactoring, use MembershipPaymentValidator instead
```

Comment:
If the class is used nowhere in this context, you can remove it - That'll make integration in the Fundraising Application code easier because we'll get hard errors instead of having to look for the deprecated class.